saas/axops/src/applatix.io/axops/configuration.go

```diff
@@ -158,58 +168,66 @@ func CreateConfiguration() gin.HandlerFunc {
 				return
 			}
 		}
-		config := &configuration.ConfigurationData{
-			ConfigurationUser:        username,
-			ConfigurationName:        name,
-			ConfigurationDesc:        description,
-			ConfigurationValue:       v,
-			ConfigurationDateCreated: time.Now().Unix(),
-			ConfigurationLastUpdated: time.Now().Unix(),
-		}
 
-		axErr = configuration.CreateConfiguration(config)
+		//Verify if config already exists
+		configList, axErr := configuration.GetConfigurationsByUserName(config.ConfigurationUser, config.ConfigurationName)
 		if axErr != nil {
 			c.JSON(axerror.REST_INTERNAL_ERR, axErr)
 			return
-		} else {
-			c.JSON(axerror.REST_CREATE_OK, common.NullMap)
+		}
+		if len(configList) != 0 {
+			c.JSON(axerror.REST_BAD_REQ, axerror.ERR_API_INVALID_REQ.NewWithMessagef("Failed to create as configuration with namespace %v and name %v already exists.",
+				config.ConfigurationUser, config.ConfigurationName))
 			return
 		}
+
+		axErr = configuration.CreateConfiguration(&config)
+		if axErr != nil {
+			c.JSON(axerror.REST_INTERNAL_ERR, axErr)
+			return
+		}
+		c.JSON(axerror.REST_CREATE_OK, common.NullMap)
+		return
 	}
 }
 
 // @Title ModifyConfiguration
 // @Description Modify configuration
 // @Accept  json
-// @Param   user             path    string   true        "user"
-// @Param   name             path    string   true        "configuration name"
-// @Param   value            body    MapType  true        "configuration value (key value paired)"
-// @Param   description      query   string   False       "configuration description"
+// @Param   configuration    body    configuration.ConfigurationData   true   "configuration object"
 // @Success 200 {object} MapType
 // @Failure 400 {object} axerror.AXError "Invalid request"
 // @Failure 500 {object} axerror.AXError "Internal server error"
 // @Resource /configurations
 // @Router /configurations/{user}/{name} [PUT]
```

Comment:
Please update swagger URL

Reply:
Actually /configurations/{user}/{name} is the right endpoint to perform modifications. I am going to change it back to original implementation.